erts/emulator/test/driver_SUITE.erl

```diff
@@ -124,7 +124,7 @@
 
 suite() ->
     [{ct_hooks,[ts_install_cth]},
-     {timetrap, {minutes, 1}}].
+     {timetrap, {minutes, 10}}].
```

Comment:
You can increase the timeout just for that specific testcase, no need to do it for the entire suite.

Reply:
Thanks, fixed and merged!